packages/edit-navigation/src/components/add-menu/index.js

```diff
@@ -101,6 +102,7 @@ function AddMenu( {
 				variant="primary"
 				disabled={ ! menuName.length }
 				isBusy={ isCreatingMenu }
+				aria-disabled={ isCreatingMenu }
```

Comment:
@Mamaduka Looking at the Button component, it looks like the existing `disabled` prop a couple of lines up sets `aria-disabled`, so I think that should be used instead of a separate `aria-disabled`. Otherwise `aria-disabled` will be `false` when `! menuName.length`.

I think the Button should still be focusable when creating a menu to avoid a focus loss, so something like this might work:
```jsx
disabled={ ! menuName.length || isCreatingMenu }
isFocusable={ isCreatingMenu }
```

Wow, Button component is confusing.

Reply:
Good catch! I wasn't aware of `__experimentalIsFocusable.`

I think we can get same result with following code and other devs won't need knowledge what `disabled` and `isFocusable` combo does under the hood.
```js
disabled={ ! menuName.length }
aria-disabled={ ! menuName.length || isCreatingMenu }
```

Reply:
Sounds good 👍 

Might be worth adding a comment near the code to explain it.

Reply:
This method is used across the core. Do we still need an inline comment?

Reply:
I still think it might be uncommon for contributors. And across Gutenberg there's probably a mixture of this and `__experimentalIsFocusable`.

Reply:
Fixed in 8b4d6d2.